django-backend/data-formatter/main.py

```diff
@@ -0,0 +1,129 @@
+'''
+
+The following file is a parser which converts the input data schema from {nodes:[], links:[]} to the distance list format.
+
+'''
+
+import shelve
+# For DeepCopy
+import copy
+# For JSON file output
+import json
+
+# INPUT DATA SOURCE FILENAME
+INPUT_FILE_NAME = 'fdg_input_file.json'
+OUTPUT_FILE_NAME = 'fdg_output_file'
+# Maximum distance to be considered
+MAX_DISTANCE = 10
+
+
+
+'''
+Builds schema for an individual node

```

Comment:
docstrings should be inside the function, not outside, see https://www.python.org/dev/peps/pep-0257/

Reply:
Thanks for sharing the link. I have incorporated the changes.